spec/src/consensus.rs

```diff
@@ -536,6 +550,8 @@ pub struct Consensus {
     pub permanent_difficulty_in_dummy: bool,
     /// A switch to select hard fork features base on the epoch number.
     pub hardfork_switch: HardForkSwitch,
+    pub well_known_lock_scripts: Vec<Script>,
+    pub well_known_type_scripts: Vec<Script>,
```

Comment:
According to my understanding, `Consensus` serves as an identifier of a blockchain. I don't understand why `well_known-*_ scripts` should be included in `Consensus`?

Reply:
I totally agree with @keroro520.

It's not a part of `Consensus`. It belongs to RPC module.

Reply:
moved the related code to `PoolRpcImpl`, please review again, thanks.